roles/kube_prometheus_stack/defaults/main.yml

```diff
@@ -12,46 +12,158 @@ kube_prometheus_release_name: kube-prometheus-stack
 # The timeout to wait for NGINX ingress controller to become ready
 # kube_prometheus_wait_timeout: 10m
 
-# The values for the release
-kube_prometheus_release_defaults:
-  alertmanager:
-    enabled: "{{ azimuth_management_cluster_monitoring_enabled }}"
-    config:
-      global:
-        # Variable should be defined in secrets file elsewhere
-        slack_api_url: "{{ kube_prometheus_stack_alertmanager_slack_url }}"
-      route:
-        receiver: 'slack-notifications'
-        group_by: ['namespace']
-      receivers:
-      - name: 'null'
-      - name: 'slack-notifications'
-        slack_configs:
-        - channel: '#general'
-          send_resolved: true
-  prometheus:
-    prometheusSpec:
-      # Should be less than storage volume size below
-      retentionSize: 9.9GB
-      storageSpec:
-        volumeClaimTemplate:
-          spec:
-            # Leave storageClassName blank to use cluster's default storage class
-            # storageClassName: <name>
-            accessModes: ["ReadWriteOnce"]
-            resources:
-              requests:
-                storage: 10Gi
-  grafana:
-    sidecar:
-      dashboards:
-        # List of namespaces in which to look for ConfigMaps with the label
-        # `grafana_dashboard: "1"` which signifies that the map contains
-        # additional dashboards to be imported into Grafana
-        searchNamespace:
-        - monitoring-system
-        - azimuth
+# Enable alerting on the Azimuth management cluster
+azimuth_management_cluster_alerting_enabled: false
+
+# Enable Slack alertmanager alerts
+kube_prometheus_stack_alertmanager_slack_alerts: false
+
+# Slack webhook URL for alertmanager alerts
+kube_prometheus_stack_alertmanager_slack_url: >- 
+  {{-
+    undef(hint = 'kube_prometheus_stack_alertmanager_slack_url is required')
+    if kube_prometheus_stack_alertmanager_slack_alerts
+    else None
+  }}
```

Comment:
Should we mention that slack alerting can be disabled via `kube_prometheus_stack_alertmanager_slack_alerts` in the hint?

Reply:
Can do, although that's set `false` by default, and hopefully people wouldn't enable the slack alerts and not expect to provide a webhook URL!

We should add an Alerting section to the azimuth-config docs to clarify I think...

Reply:
I'm happy to add details to the docs instead of here